### DISCUSSION TITLE
(Play) Store updates

### DISCUSSION THREAD

**Discussion by azrdev** [TechbeeAT/jtxBoard#1109] [Q&A] [answered]
I'm suffering https://github.com/TechbeeAT/jtxBoard/issues/610 which is apparently fixed since v2.5.0 from 2023-07 -- but google play store apparently has 2.06.04.gplay and f-droid for sure has that version. I'm just looking at f-droid, how to trigger the update there, so the question here is:

Would you update the version in gplay store? When does that happen normally / what delay is to be expected (esp. since gplay reviews can take a while AFAIK)?

**Comment by patrickunterwegs** [comment, marked as answer]
Hi @azrdev , 
I'm not sure if I understand the problem. Basically the updates on Google Play are published faster than on f-Droid. Google Play and f-Droid have 2.6.4 as the latest version. 
Are you sure that you still have an old version installed? What does the about screen say? If you use f-Droid, maybe also there's a problem with automatic updates on your store app?

**Reply by azrdev** [reply]
Hi, thanks for that prompt reply!
And sorry, apparently it's a misunderstanding on my behalf, I got confused by the leading zeroes.
So both stores have 2.06.04 which is [v2.6.4](https://github.com/TechbeeAT/jtxBoard/releases/tag/v2.6.4) -- all good!
(Except for tasks opened from the widget sometimes showing the mentioned empty screen for some seconds, until they load -- but that's a different issue)

**Reply by patrickunterwegs** [reply]
Alright, no worries, there is another fix for such an issue in the upcoming version 2.6.5 that is currently in beta on Google Play, I will probably publish it within next week!
